src/saml2/response.py

```diff
@@ -63,7 +63,7 @@ def for_me(condition, myself ):
 def authn_response(conf, return_addr, outstanding_queries=None,
                     log=None, timeslack=0, debug=0, asynchop=True,
                     allow_unsolicited=False):
-    sec = security_context(conf)
+    sec = security_context(conf, log=log)
```

Comment:
I don't see `log` as a kwarg in the implementation of security_context: http://nullege.com/codes/search/saml2.sigver.security_context